src/main/java/school/faang/bjs2_74534/QuestSystem.java

```diff
@@ -0,0 +1,33 @@
+package school.faang.bjs2_74534;
+
+import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
+
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+@Slf4j
+@Getter
+public class QuestSystem {
+    private static final long MILLIS = 1000;
```

Comment:
за константу лайк, если подушнить, то можно завести лямбду, приниммающую инт:
```suggestion
    private static final Supplier<Integer> QUEST_TIME_CALCULATOR = difficulty -> difficulty * 1000 ;
```

Reply:
Идея интересная, только я не пойму как прикрутить это, ведь у Supplier\<T\> нет параметров. А если указать как написано, то возникает ошибка "error: incompatible types: incompatible parameter types in lambda expression" и java показывает на первый "difficulty" 
Подскажи как использовать твою конструкцию с лямбдой.

Reply:
Подскажи как использовать твою конструкцию с лямбдой.

Reply:
Сорян, используй Function<Integer, Integer>

Reply:
Идея прикольная! Благодарю. Надеюсь я правильно тебя понял?